gridappsd/field_interface/interfaces.py

```diff
@@ -164,194 +165,25 @@ def unsubscribe(self, topic):
         pass
 
     @abstractmethod
-    def publish(self, data, topic: str = None):
+    def send(self, topic, data):
```

Comment:
elsewhere we use topic message....which should it be?

Reply:
Changed to message.